Doc/library/subprocess.rst

```diff
@@ -442,6 +450,17 @@ functions.
    :program:`ps`.  If ``shell=True``, on POSIX the *executable* argument
    specifies a replacement shell for the default :file:`/bin/sh`.
 
+   .. versionchanged:: 3.6
+      *executable* parameter accepts a :term:`path-like object` on POSIX.
```

Comment:
Was it added by mistake?

Reply:
It was probably added to resolve the conflict but it belongs in the 3.7 docs as well. For some reason this doc bit wasn't ported back to 3.6 and 3.7 when Serhiy landed GH-5914.

I would leave it in.